jackBot.py

```diff
@@ -151,7 +165,7 @@ def get_units(self):
         self.actions = []
 
         if iteration == 0:
-            self._client.game_step = 4  # actions every 4 frames-(optimizing so we can get it to 1 is ideal)
+            self._client.game_step = 8  # actions every 4 frames-(optimizing so we can get it to 1 is ideal)
```

Comment:
why change this? was a bug from in the old version?

Reply:
that was just for testing, i always forget to turn it back